### ISSUE TITLE
[REQ] Better markdown css styles

### ISSUE THREAD

**Issue by gingerbeardman** [glushchenko/fsnotes#263]
(If I can figure where these styles are set I will attempt this myself)

Compare...

**Ulysses**

![screen shot 2018-05-30 at 12 21 03](https://user-images.githubusercontent.com/49612/40717377-01378518-6404-11e8-877d-5343e365f153.png)

**FSNotes**

![screen shot 2018-05-30 at 12 21 50](https://user-images.githubusercontent.com/49612/40717405-1ca981d4-6404-11e8-9d81-0fc980effb41.png)

**Comment by gingerbeardman** [comment]
@glushchenko can you please advise where these styles are stored?

**Comment by clayreimann** [comment]
@gingerbeardman I think the files you're looking for are in [Resources/DownView.bundle](https://github.com/glushchenko/fsnotes/tree/master/Resources/DownView.bundle)

**Comment by gingerbeardman** [comment]
@clayreimann I thought so, but deleting the contents of the CSS file contained in that bundle sees no changes? Unless it is cached somehow?

**Comment by glushchenko** [comment]
@gingerbeardman no specific settings for this. Colors hardcoded in NotesTextProcessor 🙃

**Comment by gingerbeardman** [comment]
Oh! 😕

**Comment by TwisterMc** [comment]
I would also love to see h1s bigger than h2s and so on. I tried to change the CSS file noted above but it doesn't take effect. Maybe adding a theming engine would help this and then people can create whatever they want?

**Comment by gingerbeardman** [comment]
Maybe exposing the CSS as a note would be enough?

**Comment by TwisterMc** [comment]
As a starting point maybe. I just worry that someone might accidentally delete it.

**Comment by ment4list** [comment]
I just got this app and it's fast and great but the lack of a nicer theme is a bit of a drawback. 
I especially like how Bear handles links and the difference in heading sizes is important as well. The margin/padding between headings and text in FSNotes is also a bit minimal, which isn't too nice.

_I realize it's all subjective, but just wanted to contribute some ideas._

I agree that a theming option would be fantastic. Even if it's just a CSS file somewhere.

Here's a comparison between Bear and FSNotes on macOS:

Bear
<img width="461" alt="Bear Preview" src="https://user-images.githubusercontent.com/1780782/86735691-41eae680-c033-11ea-9d4b-574fc7f7554d.png">


FSNotes
<img width="448" alt="FSNotes Preview" src="https://user-images.githubusercontent.com/1780782/86735749-4d3e1200-c033-11ea-9162-a184dd6f2edf.png">

**Comment by gingerbeardman** [comment]
I've been discussing this with @glushchenko  privately too. 

My favourite styles are in Pretext (iOS) and Ulysses (iOS and macOS). 

I believe the theme is kind of embedded right now, and would need extracting/exposing in some way to make this doable.

**Comment by dcd-arnold** [comment]
Yeah that would be great. I never leave edit mode and it is hard to read notes like that.

**Comment by dcd-arnold** [comment]
Is there a chance this will make it into version 6?

**Comment by glushchenko** [comment]
@dcd-arnold what I should do? There is not one request, but several about creating themes.

**Comment by dcd-arnold** [comment]
I would like to be able to alter CSS for the edit mode. If you could expose it in the same way like the preview mode, that would be nice. Is that possible?

**Comment by ReessKennedy** [comment]
+1 ... Day 3 of my uages of the amazing FS Notes and I am trying to figure out how to do this too. I suspect @glushchenko likes the very stripped down edit mode maybe influenced by Notational Velocity use of the past but I feel like there is little downside to just allow editor editing as well as preview too. 

The ability to edit this would allow people to make it as simple or Bear-like as they want. The ability to create your own editor thing would be a big and new way that FS Notes gives users ownership and control over their life with their notes vs. Bear, Upnote, Simplenotes, etc ...

Perhaps just change preview.css to editor.css and then have each editor state have a class so users could write everything one file, like `.editmode { h2 { } }` and `.previewmode { h2 { } }`
